htdocs/contact/class/contact.class.php

```diff
@@ -1776,10 +1776,13 @@ public function getContactRoles($element = '')
 		$sql = "SELECT sc.fk_socpeople as id, sc.fk_c_type_contact";
 		$sql .= " FROM ".MAIN_DB_PREFIX."c_type_contact tc";
 		$sql .= ", ".MAIN_DB_PREFIX."societe_contacts sc";
+		$sql .= " INNER JOIN ".MAIN_DB_PREFIX."socpeople sp";
+		$sql .= " ON sc.fk_socpeople = sp.rowid";
 		$sql .= " WHERE sc.fk_soc =".((int) $this->socid);
 		$sql .= " AND sc.fk_c_type_contact=tc.rowid";
 		$sql .= " AND tc.element = '".$this->db->escape($element)."'";
 		$sql .= " AND tc.active = 1";
+		$sql .= " AND sp.statut = 1";
```

Comment:
When you use the INNER JOIN syntax, you must and the condition on the same line than the "ON ..." so on line 1780
$sql .= " ON sc.fk_socpeople = sp.rowid AND sp.statut = 1";

Reply:
Update OK